From 28647d9e3ce2b4ac83e922891e0462de503b8214 Mon Sep 17 00:00:00 2001
From: Stefan Seyfried <seife@tuxbox-git.slipkontur.de>
Date: Fri, 1 Jan 2016 19:31:45 +0100
Subject: [PATCH 7996/8016] opkg_manager: tone down diagnostics in badpackage()

In case of many packages, it can take a very long time to output the
matches of badpackage() on a serial console, so only print them if the
verbosity is increased and remove unnecessary information.

Signed-off-by: MarkusVolk <f_l_k@t-online.de>
---
 src/gui/opkg_manager.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gui/opkg_manager.cpp b/src/gui/opkg_manager.cpp
index 06bccf6..aa3d7f0 100644
--- a/src/gui/opkg_manager.cpp
+++ b/src/gui/opkg_manager.cpp
@@ -379,7 +379,7 @@ bool COPKGManager::badpackage(std::string &s)
 	}
 
 	if (!st.empty()){
-		dprintf(DEBUG_NORMAL,  "[COPKGManager] [%s - %d] found bad package => %s [filtered with %s]\n", __func__, __LINE__, s.c_str(), st.c_str());
+		dprintf(DEBUG_INFO, "[%s] filtered '%s' pattern(s) '%s'\n", __func__, s.c_str(), st.c_str());
 		return true;
 	}
 
-- 
2.1.4

